node/src/GlideClusterClient.ts

```diff
@@ -567,4 +568,24 @@ export class GlideClusterClient extends BaseClient {
     public time(route?: Routes): Promise<ClusterResponse<[string, string]>> {
         return this.createWritePromise(createTime(), toProtobufRoute(route));
     }
+    /**
+     * Returns the number of keys in the database.
+     *
+     * See https://valkey.io/commands/dbsize/ for more details.
+
+     * @param route - Specifies the routing configuration for the command. The client will route the
```

Comment:
You need to state where the command is routed if no route is provided (see https://github.com/valkey-io/valkey-glide/pull/1540/files#diff-e0425ed432ae9320e2081f14f090f6f5def758713e2964c6c9c8fa1fc600eea6)